src/xdocs/config_imports.xml

```diff
@@ -892,6 +892,8 @@ public class InputIllegalImport { }
               <li>Check starts checking from the longest matching subpackage (later 'current
                   subpackage') described inside import control file to package defined in class
                   file.</li>
+              <li>Order of rules in the same subpackage/root are defined by the order of
+                  declaration in the XML file.</li>
               <li>If there is matching allow/disallow rule inside the current subpackage then the
```

Comment:
Please explicitly describe that use first matched rule from top to the bottom.

Reply:
Done.